<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Minimalflaechen</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tab_widget">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="parameter_tab">
       <attribute name="title">
        <string>Einstellungen</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0">
        <item row="9" column="4">
         <widget class="QPushButton" name="quit_button">
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="4">
         <widget class="QPushButton" name="run_button">
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </item>
        <item row="7" column="4">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="3" rowspan="3">
         <widget class="QGroupBox" name="definitionsbereich">
          <property name="title">
           <string>Gebietsgröße:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5" columnstretch="1,0">
           <item row="0" column="0">
            <widget class="QLabel" name="xmin_label">
             <property name="text">
              <string>xmin:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="xmin_edit">
             <property name="text">
              <string>-0.5</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="xmax_label">
             <property name="text">
              <string>xmax:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="xmax_edit">
             <property name="text">
              <string>0.5</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="ymin_label">
             <property name="text">
              <string>ymin:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="ymin_edit">
             <property name="text">
              <string>-0.5</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="ymax_label">
             <property name="text">
              <string>ymax:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="ymax_edit">
             <property name="text">
              <string>0.5</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="7" column="0" rowspan="3" colspan="2">
         <widget class="QGroupBox" name="numerische_parameter">
          <property name="title">
           <string>Numerische Parameter:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,0">
           <item row="0" column="0">
            <widget class="QLabel" name="epsilon_label">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Fehlergrenze&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>epsilon:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="iteration_label">
             <property name="text">
              <string>iterations:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="epsilon_edit">
             <property name="text">
              <string>1e-09</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="iteration_edit">
             <property name="text">
              <string>20</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="m_edit">
             <property name="text">
              <string>30</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="n_label">
             <property name="text">
              <string>stuetz_x:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="n_edit">
             <property name="text">
              <string>30</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="m_label">
             <property name="text">
              <string>stuetz_y:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="0" colspan="4">
         <widget class="QGroupBox" name="grenzen">
          <property name="title">
           <string>Randfunktionen:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="4" column="5">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>           Gebiet</string>
             </property>
            </widget>
           </item>
           <item row="8" column="5">
            <widget class="QLineEdit" name="d_edit">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QLineEdit" name="b_edit">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="7" column="5">
            <widget class="Line" name="bottom_line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="8">
            <widget class="QLabel" name="c_label">
             <property name="text">
              <string>c:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="7">
            <widget class="Line" name="right_line">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="8" column="3">
            <widget class="QLabel" name="d_label">
             <property name="text">
              <string>d:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="a_label">
             <property name="text">
              <string>a:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="b_label">
             <property name="text">
              <string>b:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="9">
            <widget class="QLineEdit" name="c_edit">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="Line" name="top_line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="a_edit">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="4" column="4">
            <widget class="Line" name="left_line">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="6" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="console_tab">
       <attribute name="title">
        <string>Konsolenausgabe</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTextEdit" name="output_message"/>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="view_tab">
       <attribute name="title">
        <string>Anzeige</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="1" column="0">
         <widget class="Plot" name="plot_widget" native="true"/>
        </item>
        <item row="0" column="0">
         <widget class="QProgressBar" name="progressBar_plot">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEinstellungenLaden"/>
   <addaction name="actionEinstellungenSpeichern"/>
   <addaction name="actionErgebnis_laden"/>
   <addaction name="actionErgebnis_speichern"/>
  </widget>
  <action name="actionErgebnis_laden">
   <property name="text">
    <string>Ergebnis laden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionErgebnis_speichern">
   <property name="text">
    <string>Ergebnis speichern</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionEinstellungenLaden">
   <property name="text">
    <string>Einstellungen laden</string>
   </property>
  </action>
  <action name="actionEinstellungenSpeichern">
   <property name="text">
    <string>Einstellungen speichern</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plot</class>
   <extends>QWidget</extends>
   <header>plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
